sql/psql/OMERO5.3DEV__7/delete-ns-orphans.sql

```diff
@@ -0,0 +1,56 @@
+-- Copyright (C) 2016 University of Dundee. All rights reserved.
+-- Use is subject to license terms supplied in LICENSE.txt
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 2 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License along
+-- with this program; if not, write to the Free Software Foundation, Inc.,
+-- 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+--
+
+---
+--- Delete non-sharable orphaned annotations from OMERO 5.3. Optional.
+---
+
+DELETE FROM annotation WHERE
```

Comment:
Unsure.  I imagine that it would be entirely transparent, up until the point you need that bit of nonstandard SQL for the deletes.